packages/console/src/widget.ts

```diff
@@ -381,6 +399,7 @@ class CodeConsole extends Widget {
   protected onBeforeDetach(msg: Message): void {
     let node = this.node;
     node.removeEventListener('keydown', this, true);
+    node.removeEventListener('mousedown', this);
```

Comment:
Should be `click`?  Also, should it be `click` vs. `onclick` above?

Reply:
Good catch.